package.json

```diff
@@ -87,6 +87,7 @@
     "boundless-popover": "^1.0.4",
     "bugsnag": "^1.7.0",
     "bull": "^3.3.7",
+    "cancan": "3.1.0",
```

Comment:
I checked out the source code of this, it's nice and simple with a minimal API so we can move it in house without too many changes if it turns out that it's restrictive at all.